assets/js/modules/adsense/components/common/UseSnippetSwitch.js

```diff
@@ -80,17 +84,26 @@ export default function UseSnippetSwitch( props ) {
 	return (
 		<Fragment>
 			<div className="googlesitekit-setup-module__switch">
-				<Switch
-					label={ label }
-					onClick={ onChange }
-					checked={ useSnippet }
-					disabled={ isDoingSaveUseSnippet }
-					hideLabel={ false }
-				/>{ ' ' }
-				<Badge
-					className="googlesitekit-badge--primary"
-					label={ __( 'Recommended', 'google-site-kit' ) }
-				></Badge>
+				{ createInterpolateElement(
+					__( '<Switch /> <Badge/>', 'google-site-kit' ),
```

Comment:
I am not sure that this is really needed. For sites with RTL, this will be swapped automatically.

Reply:
That's a good point, this was a bit overzealous 😆